specifyweb/frontend/js_src/lib/components/SchemaConfig/Tables.tsx

```diff
@@ -85,8 +86,12 @@ export function tablesFilter(
   selectedTables: RA<keyof Tables> | undefined = undefined
 ): boolean {
   if (selectedTables?.includes(name) === true) return true;
+  const userIsAdmin =
+    userInformation.isadmin && userInformation.usertype === 'Manager';
```

Comment:
usertype is a sp6 concept - sp7 only users may not have it set
could yo ucheck only for `userInformation.isadmin`?

Reply:
ok :+1: